cluster-defns/hopsworks-head.yml

```diff
@@ -127,6 +127,6 @@ groups:
       - hopsmonitor::node_exporter
       - consul::master
       - hops::docker_registry
-#      - hops::fuse_mnt
+      - hops::fuse_mnt
```

Comment:
do you need to re-enable it on cloud installer or is just a typo?

Reply:
This is fine to have in there. The installer is really only for people who are trying out, so if they notice probs and give us feedback, then great.